jest.config.js

```diff
@@ -0,0 +1,30 @@
+/** jest.config.js */
+
+module.exports = {
+
+    collectCoverageFrom: [
+      '<rootDir>/src/app/**/*.ts',
+      '!<rootDir>/src/app/**/index.ts',
+      '!<rootDir>/src/app/**/*.module.ts'
+    ],
+  
+    coverageDirectory: 'coverage',
+  
+    coverageReporters: [
+      'lcov',
+      'text-summary'
+    ],
+  
+    testPathIgnorePatterns: [
+      '<rootDir>/coverage/',
+      '<rootDir>/dist/',
+      '<rootDir>/e2e/',
+      '<rootDir>/node_modules/',
+      '<rootDir>/src/app/*.(js|scss)'
+    ],
+  
+    testMatch: [
+      '<rootDir>/src/app/*.spec.ts',
```

Comment:
According to this configuration, tests should be located only inside of /src/app folder
I'd suggest changing it to /src

Also I see one test file in ./test folder. I guest we can remove this folder and test in scope of this pr
https://github.com/denisoby/dictionary-extension/blob/master/test/index.spec.ts